hack/make-release-artifacts.sh

```diff
@@ -25,46 +25,47 @@ if [[ ! -d "${bin_dir}" ]]; then
   exit 1
 fi
 
-krew_tar_archive="krew.tar.gz"
-krew_exe="krew.exe"
-
-# copy license
-cp -- "${SCRIPTDIR}/../LICENSE" "./${bin_dir}"
-
-# create a out/krew.exe convenience copy
-if [[ -x "./${bin_dir}/krew-windows_amd64.exe" ]]; then
-  cp -- "./${bin_dir}/krew-windows_amd64.exe" "./out/krew.exe"
+checksum_cmd="shasum -a 256"
+if hash sha256sum 2>/dev/null; then
+  checksum_cmd="sha256sum"
 fi
+checksum_sed=""
 
-# consistent timestamps for files in bindir to ensure consistent checksums
 while IFS= read -r -d $'\0' f; do
-  echo "modifying atime/mtime for $f"
-  TZ=UTC touch -at "0001010000" "$f"
-  TZ=UTC touch -mt "0001010000" "$f"
-done < <(find "${bin_dir}" -print0)
+  archive_dir="$(mktemp -d)"
+  cp "$f" "${archive_dir}"
+  cp -- "${SCRIPTDIR}/../LICENSE" "${archive_dir}"
+  archive="$(basename "$f" .exe).tar.gz"
```

Comment:
Let's extract this into a variable, because it is reused a few times.
```suggestion
  name=$(basename "$f" .exe)
  archive="${name}.tar.gz"
```